app/services/idv/phone_step.rb

```diff
@@ -62,16 +65,15 @@ def proof_address
       idv_session.idv_phone_step_document_capture_session_uuid = document_capture_session.uuid
 
       run_job(document_capture_session)
-      add_proofing_cost
```

Comment:
since the job may be asynchronous now, I moved it to the `done` check? I think this is right? @mitchellhenke would love to have the logic double-checked here

Reply:
This will work most of the time but it's still dependent on the client poll.  it would be great if we could put it in the worker :-)

Reply:
Yeah, will move inside the job to be more reliable

Reply:
Updated in 99979628f497e9896e6a1b4442a5bcd203b36b86

Reply:
You ROCK!

Reply:
needing the user_id and issuer...

Reply:
you still rock